server/filters/formatYesNoFilter.ts

```diff
@@ -0,0 +1,27 @@
+export default function formatYesNoFilter(value: string | boolean): string {
```

Comment:
Whilst this has been written to handle CIAG enum values of YES, NO and NOT_SURE; it felt like we could make it a bit more generic so that it can also handle booleans, Y and N as well 👍